modules/common/src/test/resources/iglu-client-embedded/schemas/nl.basjes/yauaa_context/jsonschema/1-0-4

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
It seems like we have a duplicate of this schema here and in `common-fs2/src/test`. Can we not use a single file? And do we need it hardcoded once the iglu-central PR is merged?

Reply:
This is the tip of something bigger.  We have [a lot of iglu central schemas](https://github.com/snowplow/enrich/tree/3.5.0/modules/common-fs2/src/test/resources/iglu-client-embedded/schemas) hardcoded into Enrich's test resources.  Several of them are duplicated between `common` and `commonFs2`.

Yes I could deduplicate those duplicates, but then it would look a bit odd that `commonFs2` has some gaps in the long list of schemas that it maintains.

> do we need it hardcoded once the iglu-central PR is merged?

I like how they are locally available when the tests run.  For unit tests especially, it is good to avoid going to the network.  But that doesn't mean they need to be hardcoded in Enrich!

One solution could be to download selected iglu central schemas before the tests run, and add them to a shared resources directory so they are available for all tests.  It could be implemented as a sbt task, so that `test` depends on this new task.  It would provide a better guarantee that our code is valid against the real Iglu Central.

What do you think?

Reply:
I think that would be great! It exposes us to some rogue iglu central patch; but that seems less likely than us forgetting to update a schema somewhere in the hardcoded files.

Reply:
Idea for the future, but I will create an issue now.